Problems with latexpdf ing the documentation

If I try to build the documentation as described in the [Readme](https://github.com/JuliaLang/julia/tree/master/doc),
```
sudo apt-get install latex-cjk-all texlive texlive-lang-cjk texlive-latex-extra
make latexpdf
```
I get the following error:
```
! Package inputenc Error: Unicode char ∓ (U+2213)
(inputenc)                not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.29842 ...e the scaled Hankel function \(\exp(∓
                                                  iz) H_ν^{(k)}(z)\), where...

? ^Cmake[1]: *** Deleting file 'JuliaLanguage.pdf'
! Interruption.
\GenericError  ...                                
                                                  \endgroup 
l.29842 ...e the scaled Hankel function \(\exp(∓
                                                  iz) H_ν^{(k)}(z)\), where...
```

I tried this on a fresh Ubuntu16.04 with the following julia:

Julia Version 0.5.0-rc0+0
Commit 0030eec (2016-07-26 20:22 UTC)
Platform Info:
  System: Linux (x86_64-linux-gnu)
  CPU: Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, haswell)

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -201,6 +201,7 @@ latex_elements = {
         \DeclareUnicodeCharacter{025B}{\ensuremath{\varepsilon}}
         \DeclareUnicodeCharacter{03B3}{\ensuremath{\gamma}}
         \DeclareUnicodeCharacter{03B8}{\ensuremath{\theta}}
++       \DeclareUnicodeCharacter{03BD}{\ensuremath{\nu}}
         \DeclareUnicodeCharacter{03C0}{\ensuremath{\pi}}
         \DeclareUnicodeCharacter{03C6}{\ensuremath{\varphi}}
         \DeclareUnicodeCharacter{1D34}{\ensuremath{^{\mathrm{H}}}}
@@ -213,6 +214,7 @@ latex_elements = {
         \DeclareUnicodeCharacter{2209}{\ensuremath{\notin}}
         \DeclareUnicodeCharacter{220C}{\ensuremath{\not\ni}}
         \DeclareUnicodeCharacter{2211}{\ensuremath{\sum}}
++       \DeclareUnicodeCharacter{2213}{\ensuremath{\mp}}
         \DeclareUnicodeCharacter{221A}{\ensuremath{\sqrt{}}}
         \DeclareUnicodeCharacter{221B}{\ensuremath{\sqrt[3]{}}}
         \DeclareUnicodeCharacter{222A}{\ensuremath{\cup}}

